### ISSUE TITLE
[AirbnbRating] The item which is unrated will lose its background color

### ISSUE THREAD

**Issue by wangyi7099** [Monte9/react-native-ratings#133]
![111](https://user-images.githubusercontent.com/24426655/103967337-bd715000-519c-11eb-8a56-e9eb5bb0ba9f.png)
Hi This is my code
```html
<AirbnbRating
    reviewColor="#cecece"
    defaultRating={0}
    count={4}
    showRating
    reviews={['1', '2', '3', '4']}
    style={{paddingVertical: 10}}
  />
```

**Comment by gohilmukund** [comment]
I am also facing the same issue. Happens only in Android while it is working in IOS.

**Comment by CarlosLeonCode** [comment]
Yes, I've the same situation...

**Comment by wangyi7099** [comment]
My solution is copy the all files from node_modules to the fold under the root of my repo and  modify `start.js`
```jsx
<TouchableOpacity activeOpacity={1} onPress={this.spring.bind( this )} disabled={isDisabled}>
        <Animated.Image
          source={starSource}
          style={[
            styles.starStyle,
            {
              tintColor: fill   ? selectedColor : "#cecece", // I changed here
              width: size || STAR_SIZE,
              height: size || STAR_SIZE,
              transform: [{ scale: this.springValue }]
            },
            starStyle
          ]}
        />
      </TouchableOpacity>
```

**Comment by CarlosLeonCode** [comment]
Thanks, that works!
<http://127.0.0.1:5500/temp/sign.html#> *Carlos Esteban León*

Software developer full stack
<https://linkedin.com/in/carlos-esteban-le%C3%B3n-pinilla-a646a9125>
<https://github.com/CarlosLeonCode> <https://gitlab.com/CarlosCode>
+57 3186199380Bogotá, ColombiaWebsite
<https://carlos-esteban-leon-pinilla.netlify.app/>

**Comment by tomy-pg** [comment]
This is an Android specific bug. I am also experiencing the same thing.

**Comment by rahulbhankar786** [comment]
@wangyi7099 is there any permanent solution for that

**Comment by sshivananda** [comment]
Fixed in v7.4.0
